go.mod

```diff
@@ -5,6 +5,7 @@ go 1.14
 replace (
 	git.apache.org/thrift.git => github.com/apache/thrift v0.12.0
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible
+	github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.39.0
```

Comment:
why do we need this in the replace block? can we directly update line 56

Reply:
We could directly update it, but I believe it would require changes over a number of different files that import it. Since it's just a name change of the same underlying repository, I thought it made more sense to use a replace block instead of propagating those changes to files that can be left untouched.

I don't feel too strongly about it though, so if it would be better to make those changes I can re-run go generate

Reply:
Couple of comments:
1. Why not just fixup the imports? I think that is the better option, if the upstream changed we should just change as well. 
2. Why are we updating to such an old version? 0.44.1 is the most recent, vs 39 which was released in may

Reply:
@dramich I can update to 0.44.1, but I was trying to make the minimal update required for the original issue with the Monitoring v0.2.0 chart (which is just going to version 0.39.0).

Will update in the next commit, and will fixup the imports when I run go generate this time.